ui-v2/tests/acceptance/dc/acls/policies/update.feature

```diff
@@ -35,8 +40,8 @@ Feature: dc / acls / policies / update: ACL Policy Update
       ------------------------------------------------------------------------------
       | Name          |  Rules                        | Description                |
       | policy-name   |  key "foo" {policy = "read"}  | policy-name description    |
-      | policy_name   |  key "foo" {policy = "write"} | policy name description    |
-      | policyName    |  key "foo" {policy = "read"}  | policy%20name description  |
+      # | policy_name   |  key "foo" {policy = "write"} | policy name description    |
```

Comment:
Any reason these were removed? It didn't seem like relevant bits were changed?

Reply:
Oh good spot! These shouldn't be commented will fix, thankyou!